ts/nni_manager/core/nnimanager.ts

```diff
@@ -422,6 +391,40 @@ class NNIManager implements Manager {
         return this.dataStore.listTrialJobs(status);
     }
 
+    private get maxDuration(): number {
+        const value = this.experimentProfile.params.maxExperimentDuration;
+        return (value === undefined ? Infinity : toSeconds(value));
+    }
+
+    private get maxTrialNum(): number {
+        const value = this.experimentProfile.params.maxTrialNumber;
+        return (value === undefined ? Infinity : value);
+    }
+
+    private async initTrainingService(config: ExperimentConfig): Promise<TrainingService> {
+        this.config = config;
+        const platform = Array.isArray(config.trainingService) ? 'hybrid' : config.trainingService.platform;
+
+        if (['remote', 'pai', 'aml', 'hybrid'].includes(platform)) {
```

Comment:
Set reuse mode as default for `remote` and `pai`?

Reply:
done